profiler/profiler.go

```diff
@@ -192,6 +194,21 @@ func newProfiler(opts ...Option) (*profiler, error) {
 	if cfg.logStartup {
 		logStartup(cfg)
 	}
+	var tags []string
+	for _, tag := range cfg.tags.Slice() {
+		// If the user configured a tag via DD_VERSION or WithVersion,
+		// override any version tags the user provided via WithTags,
+		// since having more than one version tag breaks the comparison
+		// UI
+		if cfg.version != "" && strings.HasPrefix(tag, "version:") {
```

Comment:
Q: Do we need to worry about uppercase here?

Reply:
I hadn't considered this. It looks like tags to get normalized somewhere. I tried uploading a profile with `mytag:foo` and `MyTag:bar`, and searching for it only turned up anything with either `mytag:foo` or `mytag:bar` (lower case). So I think it'd be good to be case-insensitive here 👍